### ISSUE TITLE
Login node gateway issue

### ISSUE THREAD

**Issue by aaronhcarr** [xcat2/xcat-core#6897]
I have a login node in an xCAT cluster.

The idea is that the master node and login nodes are the only nodes that have an outside network connection.

I added the following definitions to the login node:
chdef login01 nictypes.em4=ethernet
chdef login01 nicips.em4=xxx.xxx.xxx.xxx
chdef login01 nicnetworks.em4=externalnet
chdef login01 routenames="defaultroute,externalnet"

em4 is the outside interface.  It is connected, does have link, the port is functioning correctly (validated by swapping cable with master node), however, a user cannot ssh directly to the login node.

I assumed that "chdef login01 routenames="defaultroute,externalnet"" was going to cause the login node to use em4 as the default route.  But it doesn't seem to be doing that.

What did I do wrong?

Thanks.

**Comment by gurevichmark** [comment]
@aaronhcarr Have you been able to follow these instructions for setting up routing:

https://xcat-docs.readthedocs.io/en/stable/guides/admin-guides/manage_clusters/ppc64le/diskful/customize_image/network/cfg_routes.html

**Comment by aaronhcarr** [comment]
I thought I did, but also, I notice that the default route for the internal network is still present.

So I was thinking that maybe I just need to get rid of that using this:
makeroutes login01 -d -r internalnet
updatenode login01 -P confignetwork

Thoughts?

**Comment by gurevichmark** [comment]
Yes, you can try that. Also check if `externalnet` appears in the networks table.

**Comment by aaronhcarr** [comment]
So I've tried some more things.

Defined the internalnet default route and the externalnet default route.
chdef -t route defaultroute net=172.16.0.0 mask=255.255.252.0 gateway=172.16.0.1 ifname=em1
chdef -t route externaltroute net=externalnet mask=255.255.255.0 gateway=192.168.1.1 ifname=em4

Added those routes to the node:
makeroutes login01 -r defaultroute,externaltroute

Deleted the defaultroute from the node:
makeroutes login01 -d -r defaultroute

Updated the network on the node:
updatenode login01 -P confignetwork


The default route is still present:
[root@login01 ~]# route
Kernel IP routing table
Destination     Gateway         Genmask         Flags Metric Ref    Use Iface
default         xcat-master.del 0.0.0.0         UG    0      0        0 em1
link-local      0.0.0.0         255.255.0.0     U     1002   0        0 em1
172.16.0.0      0.0.0.0         255.255.252.0   U     0      0        0 em1

If I manually set them via cli, everything works fine.
route del default gw 172.16.0.1
route add default gw 192.168.1.1 em4

What am I missing here?  Those 2 things are basically all I want to define to xCAT for the login node.  Delete the internalnet default gw that says use 172.16 and em1, and add the externalnet default gateway out of em4.
